Scheduler/Task.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Scheduler
+{
+	/// <summary>
+	/// Task Table Info
+	/// </summary>
```

Comment:
I think this comment is redundant.
Comments is needed when you create some specific logic and it's not clear for the other programmers why this logic implemented this way. So in this case comment should be for explanation.